settings.gradle

```diff
@@ -1,16 +1,16 @@
 pluginManagement {
     plugins {
-        id "com.diffplug.spotless" version "5.7.0"
-        id "com.github.ben-manes.versions" version "0.20.0"
+        id "com.diffplug.spotless" version "5.8.2"
+        id "com.github.ben-manes.versions" version "0.36.0"
         id "com.github.kt3k.coveralls" version "2.0.1"
-        id "com.google.protobuf" version "0.8.8"
-        id "com.jfrog.artifactory" version "4.13.0"
-        id "com.jfrog.bintray" version "1.8.4"  // Need *specific* version.
```

Comment:
I don't know what the comment is referring to, I figure this is a safe upgrade.

Reply:
let's hope!